### DISCUSSION TITLE
r/china-travel-layoff-trip-day6-yixian-kecun

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#78] [Announcements]
# r/china-travel-layoff-trip-day6-yixian-kecun

这里是皖南红色第一村

https://blog.ops-coffee.cn/r/china-travel-layoff-trip-day6-yixian-kecun

**Comment by Liudon** [comment]
大佬，请教一下，你是用的Cloudflare自选节点吗？

**Reply by ops-coffee** [reply]
默认

**Comment by shuihunX** [comment]
我为什么要写？
记录生活，和自己对话，寻找自己，认清自己，探索自己！
